cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -193,6 +193,7 @@ data TransactionCmd
       TxShelleyWitnessCount
       TxByronWitnessCount
   | TxGetTxId (Either TxBodyFile TxFile)
+  | TxView    (Either TxBodyFile TxFile)
```

Comment:
We usually avoid vertical alignment (eg `(Either`), because if `TxGetTxId` changes length the spacing for the `TxView` will need to be changed as well.

Reply:
I don't see any bad with it, but ok.

Reply:
I've noticed vertical alignment in the code around, so I made this. Is there is a rule against vertical alignment?

Reply:
We try to avoid vertical alignment. Please see `STYLE.md`

Reply:
done